### DISCUSSION TITLE
求 LaTeX 支持！

### DISCUSSION THREAD

**Discussion by AlainOUYANG** [doocs/md#9] [Q&A] [answered]
有没有可能会支持 LaTeX 呀？现在每次都只能先发在知乎上，再把知乎生成的图片复制粘贴到公众号上。。

**Comment by yanglbme** [comment]
@AlainOUYANG 你好，目前正在分支上全面改造项目，之后会支持 LaTex

**Comment by xindoo** [comment]
> @AlainOUYANG 你好，目前正在分支上全面改造项目，之后会支持 LaTex

何时上线？

**Comment by pujfei** [comment]
如果支持LaTeX，就打败另一个付费的项目Markdown Nice了。

**Comment by yanglbme** [comment, marked as answer]
忘回复了，latex 已经支持了

**Reply by pujfei** [reply]
那太好了，我后面试试
